src/converters/BindingConverters.ts

```diff
@@ -46,7 +46,9 @@ function isBindingDirection(input: string | undefined): boolean {
 export function convertKeysToCamelCase(obj: any) {
   var output = {};
   for (var key in obj) {
-      let value = fromTypedData(obj[key]) || obj[key];
+      let valueFromDataType = fromTypedData(obj[key]);
+      // Only "undefined" will be replaced with origin object property.
```

Comment:
by "origin", did you mean "original"? Just making sure :)

Reply:
fixed